example-playbooks/kernel_version_enforcer/ubuntu/stop_reboot_start.yml

```diff
@@ -0,0 +1,122 @@
+---
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: Check if Scylla is installed
+  ansible.builtin.set_fact:
+    scylla_installation="{{ true if ansible_facts.services['scylla-server.service'] is defined else false }}"
+
+- name: Stop Scylla
```

Comment:
We should stop duplicating "Scylla Rolling Restart" versions and have one that is used everywhere.
The best version of Scylla shutdown can be found in https://github.com/scylladb/scylla-ansible-roles/blob/master/example-playbooks/rolling_ops/rolling_custom_build_install.yml.

We should add the missing "rescue" block to https://github.com/scylladb/scylla-ansible-roles/blob/master/example-playbooks/rolling_ops/rolling_restart.yml and then it would be good to be used everywhere.

Reply:
This stop and start functionality, is indeed preferably implemented only once (DRY principle). 
Perhaps as a task in `ansible-scylla-node` role, and perhaps it even makes sense to group such tasks in a folder. There is already such a task present `start_one_node.yml`.